examples/external-lib-example/app.py

```diff
@@ -0,0 +1,50 @@
+from spin_sdk import http   
+from spin_sdk.http import Request, Response
+
+# Import the Router from the upstream project http_router
+# See docs: https://github.com/klen/http-router
+from http_router import Router
+
+# This is the built-in URL parser.
+from urllib.parse import urlparse, parse_qs
+
+# You want the router declared outside of the handler function for scope.
+# the trim_last_slash means that `/foo/` and `/foo` will be treated as the
+# same.
+router = Router(trim_last_slash=True)
+
+@router.route("/")
+def handle_index(uri, request):
```

Comment:
Can we add type annotations to these functions for the parameters and return values?

Reply:
Done